README.md

```diff
@@ -316,6 +316,31 @@ monday.execute("notice", {
 });
 ```
 
+#### Open files preview dialog
+Opens a popup card with the preview of an asset
+
+**type**
+`'openFilesDialog'`
```

Comment:
![image](https://user-images.githubusercontent.com/17691874/115733528-a80de380-a391-11eb-8a20-bffe431da55c.png)

we currently support this 
this means you'll get the viewer with all of the files in the PULSE

app should live on top of board/view/item (of the files passed) - meaning have the boardEntities of the board and its assets)

if you want to show only a single image/pass a list of images regardless of their pulse (but same board), we just need to change the context upon the logic in files-preview-logic-connector.js (or add new logic) - very easy